pkg/accounts/model/follow.ts

```diff
@@ -0,0 +1,63 @@
+import type { ID } from '../../id/type.js';
+import type { AccountID } from './account.js';
+
+export interface CreateAccountFollowArgs {
+  fromID: ID<AccountID>;
+  targetID: ID<AccountID>;
+  createdAt: Date;
+  deletedAt: Date | undefined;
```

Comment:
```suggestion
  deletedAt?: Date;
```
スタイルガイドによると, こう定義したほうがよさそうです